pages/k8s/how-to-cni.md

```diff
@@ -0,0 +1,40 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Using a CNI"
+  description: How to deploy and use CNI charms
+keywords: CNI, networking, sr-iov, sriov
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: how-to-cni.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+Charmed Kubernetes supports a wide variety of network options for your cluster,
+provided by additional charms. Their use is described in the pages linked below:
+
+- [Flannel](/kubernetes/docs/cni-flannel)
+- [Calico](/kubernetes/docs/cni-calico)
+- [Canal](/kubernetes/docs/cni-canal)
+- [Cilium](/kubernetes/docs/cni-cilium)
+- [Kube OVN](/kubernetes/docs/cni-kube-ovn)
+- [Tigera Secure EE](/kubernetes/docs/tigera-secure-ee)
+- [Multus](/kubernetes/docs/cni-multus)
+- [SR-IOV](/kubernetes/docs/cni-sriov)
+- [Using multiple networks](/kubernetes/docs/multiple-networks)
+- [IPv6](/kubernetes/docs/ipv6)
+- [Ingress](/kubernetes/docs/ingress)
```

Comment:
These last three pages are networking related, but I wouldn't really call them CNI options, given that CNI is more specifically about Container/Pod networking.

I guess a quick fix would be to change this page from "How to configure CNI" to "How to configure networking".